### ISSUE TITLE
Cannot get the same camera pose result as in the paper

### ISSUE THREAD

**Issue by BiancaBing** [yinyunie/Total3DUnderstanding#33]
Hi Yinyu:
I am now trying to train the layout estimation model but cannot get the same pitch and roll error as in the paper. Also I have tested the pretrained model that is provided in your repository and I got **pitch error = 6.3016, roll error = 4.2779** that is not so good as in the paper (**pitch error = 3.15, roll error = 2.09**). I also tried the same training process as described in the paper with batch size of 32 and lr at 1e-3 (scaled by 0.5 for every 20 epochs) and in the coop repository with batch size of 32 and lr at 1e-4 ( scaled by 0.7 for every 10 epochs and finally remains at 1e-5). However, the best result I got is **pitch error = 0.56497, roll error = 0.39949**. I have no idea what happened. 
Also, the loss curve of test loss starts to increase only after a few epochs as shown below (accuracy curve here indicates the average pitch error, roll error and the sum of pitch & roll):
![cd410e60b09d88e896df0487bca0cd7](https://user-images.githubusercontent.com/42990499/115656032-c961d600-a377-11eb-95b9-2bfed575aa5f.png)
![ea6caa99d3cc33ebc8f05b70320c44e](https://user-images.githubusercontent.com/42990499/115656036-cc5cc680-a377-11eb-93d0-dd765966c7fe.png)
![5d4be108c30ba767d1653588a8ab7b4](https://user-images.githubusercontent.com/42990499/115656042-cebf2080-a377-11eb-9cbb-2d0a63c62132.png)
How can I get the same performance as in the paper?

**Comment by BiancaBing** [comment]
Hi, 

Thank you for your advice. I have tried the code from CooP and got the results. However, I still have some questions. First of all, I found the BIN value you use is different from CooP. It seems that you multiply the BIN by pi/180. Also, the test and train list you use is also different from CooP. Can I ask which benchmark you used when comparing the results in your paper?
